om4labs/diags/moc/moc.py

```diff
@@ -128,13 +128,22 @@ def read(dictArgs, varname="vmo"):
     infile = dictArgs["infile"]
     ds = xr.open_mfdataset(infile, combine="by_coords")
 
-    dsQ = horizontal_grid(dictArgs, point_type="q")
-    geolon_c = dsQ.geolon.values
-    geolat_c = dsQ.geolat.values
-    yq = dsQ.nominal_y.values
-    basin_code = dsQ.basin.values
+    # vmo
+    arr = ds[varname].to_masked_array()
+
+    outputgrid = "nonsymetric"
+    dsV = horizontal_grid(dictArgs, point_type="v", outputgrid=outputgrid)
+    if arr.shape[-2] == (dsV.geolat.shape[0] + 1):
```

Comment:
@raphaeldussin - should this check be made more general and promoted to `om4common.py`?

@adcroft and @raphaeldussin - is there a better way to detect a symmetric grid?  This currently involves generating the grid twice.  It'd be better to somehow detect a symmetric grid before calling `horizontal_grid()`